testing/environments/docker/kafka/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM spotify/kafka
```

Comment:
Yeah, probably we could add a testing/environment compose file for kafka and redis that uses these for when you want to do manual testing.